doc/classes/UndoRedo.xml

```diff
@@ -102,6 +102,8 @@
 		<method name="clear_history">
 			<return type="void">
 			</return>
+			<argument index="0" name="increase_version" type="bool" default="true">
+			</argument>
 			<description>
 				Clear the undo/redo history and associated references.
```

Comment:
Would be good to mention what `[code]increase_version[/code]` does in the description.